syncmaster/backend/api/v1/runs.py

```diff
@@ -120,7 +117,7 @@ async def start_run(
             type=RunType.MANUAL,
         )
 
-        log_url = Template(settings.worker.LOG_URL_TEMPLATE).render(
+        log_url = Template(worker_settings.LOG_URL_TEMPLATE).render(
```

Comment:
But why

Reply:
What makes `worker_settings` a global object which cannot be altered other that changing env variables before starting Python interpreter